lib/smart_proxy_ansible/reader_helper.rb

```diff
@@ -25,9 +25,9 @@ def path_from_config(config_key)
             line =~ /^\s*#{config_key}/
           end
         rescue Errno::ENOENT, Errno::EACCES => e
```

Comment:
Perhaps we should treat `EACCES` as a warning while `ENOENT` is an info?

Reply:
I thought about that, but then opted in to not making the code *too* complicated/branched.

Will see that I update it.